app/assets/javascripts/admin/modules/external_link_tracker.js

```diff
@@ -0,0 +1,35 @@
+// based on https://github.com/alphagov/govuk_frontend_toolkit/blob/master/javascripts/govuk/analytics/external-link-tracker.js
+;(function (global) {
+  'use strict'
+
+  var $ = global.jQuery
+  var externalLinkTracker = function () {
+    var externalLinkSelector = 'a[href^="http"]:not(a[href*="' + global.location.hostname + '"])'
+    $('body').on('click', externalLinkSelector, trackClickEvent)
+
+    function trackClickEvent (evt) {
+      var $link = getLinkFromEvent(evt)
+      var href = $link.attr('href')
+      var linkText = $.trim($link.text())
+      var options = {}
+
+      if (linkText) {
+        options.label = linkText
+      }
+
+      GOVUKAdmin.trackEvent('external-link-clicked', href, options);
+    }
+
+    function getLinkFromEvent (evt) {
+      var $target = $(evt.target)
+
+      if (!$target.is('a')) {
+        $target = $target.parents('a')
```

Comment:
This seems really unlikely, but is there any chance this could ever return more than one element, which could potentially break the rest of the code? For that to happen you'd need to have nested a link within a link, which would be odd, so I'm probably just being paranoid. But you could do $target.parents('a').first or something?

Reply:
It hasn't been a problem for the version running on www.gov.uk